_sass/libs/rivet-uits/utilities/_utilities-flex.scss

```diff
@@ -0,0 +1,321 @@
+// rivet-uits - @version 1.3.0
```

Comment:
This file does not have the BSD-3-Clause

Reply:
That's part of rivet. It should. Should bring it up in their repo. They have a script do it for them so I don't know why it wouldn't though... kinda weird.
I guess I can just copy one over.

Reply:
Copied it over.